.github/workflows/declare_schema_build.yml

```diff
@@ -0,0 +1,44 @@
+---
+on: [push]
+
+name: DeclareSchema Build
+
+jobs:
+  build:
+    name: DeclareSchema Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ 2.4.5, 2.5.8, 2.6.5, 2.7.1 ]
+        gemfile: [ gemfiles/rails_4.gemfile, gemfiles/rails_5.gemfile, gemfiles/rails_6.gemfile ]
+        exclude:
+          - { gemfile: gemfiles/rails_4.gemfile, ruby: 2.7.1 }
+          - { gemfile: gemfiles/rails_5.gemfile, ruby: 2.4.5 }
+          - { gemfile: gemfiles/rails_6.gemfile, ruby: 2.4.5 }
+    env:
+      BUNDLE_GEMFILE: "${{ matrix.gemfile }}"
+    steps:
+      - name: Checkout Branch
+        id: checkout_branch
+        uses: actions/checkout@v2
+      - name: Setup Ruby
+        id: setup_ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler: 1.17.3
+          ruby-version: ${{matrix.ruby}}
+      - name: Remove Bundler 2
+        id: remove_bundler_2
+        if: ${{ matrix.ruby >= '2.6.5' }}
+        run: |
+          rm -f /opt/hostedtoolcache/Ruby/2.*/x64/lib/ruby/gems/2.*/specifications/default/bundler-2.*.gemspec
+          gem install bundler:1.17.3 --force --default
+          gem install bundler -v 1.17.3
+      - name: Appraisals
+        id: appraisals
+        run: |
+          bundle install --jobs=3 --retry=3 --path=${BUNDLE_PATH:-vendor/bundle} --gemfile=${{ matrix.gemfile }}
+          git config --global user.email "example@example.com"
```

Comment:
When I remove these git config lines, I get a "Please tell me who you are" prompt that says to set these up. This wasn't necessary in the Travis file, any ideas on removing these, or do they feel fine since it's fake information?

Reply:
Hmm, this does seem like a weird work-around. Which git commands need to work? Do you have git credentials set up?

Reply:
The git.config prompt comes after this line runs in the rake task: `git init && git add . && git commit -m "initial commit"`.
I just have `id_rsa.pub` and `id_rsa` set up, do I need to have something else set up specific to declare_schema?